setup.cfg

```diff
@@ -32,10 +32,18 @@ exclude =
 
 [flake8-rst]
 bootstrap =
-    import pandas as pd
     import numpy as np
+    import pandas as pd
+    np  # avoiding error when importing again numpy or pandas
+    pd  # (in some cases we want to do it to show users)
 ignore = E402,  # module level import not at top of file
          W503,  # line break before binary operator
+         # Classes/functions in different blocks can generate those errors
```

Comment:
@datapythonista why are you adding these? we *want* to error these, yes?

Reply:
Not in the documentation. `flake8-rst` merges all the blocks in a single file to validate it. Meaning that if I have a block of code that shows a class definition, if we validate for those errors, we will get an error, as the class definition will be immediately after the last statement of the previous block.

Reply:
ok thanks. maybe add what you just wrote in a future PR as a reader would have no idea how this works.

Reply:
I tried to explain in the comment before skipping the errors, but obviously not clear enough. Will add more context/info in a future PR.